beluga_demo_nav2/launch/gazebo_launch.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import (
+    DeclareLaunchArgument,
+    SetEnvironmentVariable,
+    ExecuteProcess,
+)
+from launch.conditions import IfCondition
+from launch.substitutions import LaunchConfiguration, PythonExpression
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    # Get the launch directory
+    nav2_bringup_dir = get_package_share_directory('nav2_bringup')
+
+    # Launch configuration variables specific to simulation
+    headless = LaunchConfiguration('headless')
+    pose = {
+        'x': LaunchConfiguration('x_pose', default='-2.00'),
+        'y': LaunchConfiguration('y_pose', default='-0.50'),
+        'z': LaunchConfiguration('z_pose', default='0.01'),
+        'R': LaunchConfiguration('roll', default='0.00'),
+        'P': LaunchConfiguration('pitch', default='0.00'),
+        'Y': LaunchConfiguration('yaw', default='0.00'),
+    }
+
+    # TODO(alon): The path of this environment variable is used only by
+    # this docker. In general the real path should be:
+    # $GAZEBO_MODEL_PATH:/opt/ros/<ros2-distro>/share/turtlebot3_gazebo/models
+    gazebo_model_path_envvar = SetEnvironmentVariable(
+        'GAZEBO_MODEL_PATH',
+        '$GAZEBO_MODEL_PATH:/home/developer/ws/src/external-deps/'
+        'turtlebot3_simulations/turtlebot3_gazebo/models',
+    )
```

Comment:
@hidmic, this is the only thing that is bothering me in this package. I did that because I'm working on the Docker container created in this repo. I didn't have time to create a different Dockerfile and to test this by installing Beluga and TurtleBot3 from binaries. I'm just leaving this comment to notify you about the status of this PR.

Reply:
Hmm. Thanks for the heads up. This is a no-go. Do we need it if we are using models installed from binaries?

Reply:
And that's because `nav2_bringup` doesn't properly export `GAZEBO_MODEL_PATH`. OK with the limitation, OK with adding to the launch but pointing to `/opt/ros/humble`, not `/home/developer`.